test/addons-napi/test_instanceof/test.js

```diff
@@ -74,7 +74,8 @@ if (typeof Symbol !== 'undefined' && 'hasInstance' in Symbol &&
   compareToNative(x, MySubClass);
   compareToNative(y, MySubClass);
   compareToNative(x, MyClass);
-  compareToNative(y, MyClass);
+  // TODO: https://github.com/nodejs/abi-stable-node/issues/236
```

Comment:
@kfarnung I'm not sure about that. These are issues that we want to fix in Node-ChakraCore, not engine-specific behavior that we want to allow indefinitely.